Formula/bsdrpcgen.rb

```diff
@@ -0,0 +1,20 @@
+class Bsdrpcgen < Formula
```

Comment:
rpcgen is a part of FreeBSD base system, so it's not released separately. It's not being actively developed though, so I'd say it's stable.

Anyway, this snapshot is one commit behind from what's currently available here: https://github.com/freebsd/freebsd/commits/master/usr.bin/rpcgen

This one commit is cosmetic though, so doesn't feel like worth re-rolling.